game-core/src/main/java/games/strategy/triplea/delegate/BattleExtendedDelegateState.java

```diff
@@ -10,11 +10,14 @@
   BattleTracker battleTracker = new BattleTracker();
   public boolean needToInitialize;
   boolean needToScramble;
+  boolean needToCreateRockets;
   boolean needToKamikazeSuicideAttacks;
   boolean needToClearEmptyAirBattleAttacks;
   boolean needToAddBombardmentSources;
+  boolean needToFireRockets;
   boolean needToRecordBattleStatistics;
   boolean needToCheckDefendingPlanesCanLand;
   boolean needToCleanup;
+  RocketsFireHelper rocketHelper = null;
```

Comment:
The `null` field initializer is redundant.

```suggestion
  RocketsFireHelper rocketHelper;
```